Makefile

```diff
@@ -99,3 +99,52 @@ deploy-only: preflight-dev-kubeconfig $(tools/kubectl) build-output/yaml-$(patsu
 	@printf "$(GRN)Your ambassador image:$(END) $(BLD)$$($(tools/kubectl) --kubeconfig $(DEV_KUBECONFIG) get -n ambassador deploy ambassador -o 'go-template={{(index .spec.template.spec.containers 0).image}}')$(END)\n"
 	@printf "$(GRN)Your built image:$(END) $(BLD)$$(sed -n 2p docker/$(LCNAME).docker.push.remote)$(END)\n"
 .PHONY: deploy-only
+
+##############################################
```

Comment:
Where is all this telepresence stuff getting pulled in from?

Reply:
I left it in due to the fact it would match up with what is currently on master

Reply:
Ok, yeah I reviewed them and they don't hurt and will make back porting other changes in the future easier.